src/pm/hydra/proxy/pmip_cb.c

```diff
@@ -981,11 +984,6 @@ static HYD_status launch_procs(struct pmip_pg *pg)
                 MPL_free(str);
             }
 
-            if (pg->hostname) {
-                status = HYDU_append_env_to_list("PMI_HOSTNAME", pg->hostname, &force_env);
-                HYDU_ERR_POP(status, "unable to add env to list\n");
-            }
```

Comment:
bad rebase?

Reply:
Good catch! I will reset this after the current testing finish.